docker-centos/init.sh

```diff
@@ -18,7 +18,7 @@ do
     $i &
 done
 
-/usr/bin/docker-containerd-current \
+NOTIFY_SOCKET=/dev/null /usr/bin/docker-containerd-current \
```

Comment:
@giuseppe it might be helpful to add a comment above the line with that information. A month from now I'd probably ``git blame`` this line and then find the author so I could figure out what was going on 😸